bans-core/src/main/resources/lang/messages_es.yml

```diff
@@ -286,44 +276,104 @@ misc:
       DAYS: '%VALUE% días'
 
 
-  unknown-error: '&cUn error raro sucedio.'
+  unknown-error: '&cSe ha producido un error desconocido.'
    # Only applicable if synchronous enforcement strategy is DENY in the main config
   sync-chat-denial-message: '&cChat sincronico detenido, &7Prueba de nuevo.'
 
+admin:
+  importing:
+    failure: '&cImportación fallida. Mira la consola para ver los detalles.'
+    started: '&7Importación iniciada. Mira la consola para ver el progreso.'
+    complete: '&7Importación completada.'
+     # To prevent mistakes, it is not allowed to import multiple times at once.
+    in-progress: '&cYa hay una importe en ejecución.'
+    usage: '&cUso: /libertybans import <advancedban|litebans|vanilla>'
+
+  no-permission: '&cDisculpa, no puedes usar esto.'
+  reloaded: '&aRecargado'
+  ellipses: '&a...'
+  restarted: '&aReiniciando'
+
+ # Messages for alt-checks and alt account notifications
+ # 
+ # Before configuring this section, it is necessary to look at the address-enforcement
+ # settings in the main config.yml and understand the different kinds of alt detection.
+ # There is normal and strict detection.
+alts:
+   # Regarding the /alts command
+  command:
+    permission: '&cNo puedes revisar cuentas alternas.'
+     # The message to display atop the alt check. Set to an empty string to disable
+    header:
+      - '&7Reporte de cuentas alternas de &c&o%TARGET%.&f'
+      - '&7Alta probabilidad - Misma direccion IP que el jugador sancionado.&f'
+      - '&7Mera posibilidad - Cuenta vinculada a una dirección IP común pasada'
+    usage: '&cUso: /alts &e<jugador>&c.'
+    none-found: '&7No se han encontrado cuentas alternas'
+
+  formatting:
+     # The description for an alt account detected by strict detection.
+    strict: '&eAlta probabilidad'
+     # The description for an alt account detected by normal detection.
+    normal: '&cMera posibilidad'
```

Comment:
I believe the translations for `strict` and `normal` should be swapped. A normal detection has a strong/high possibility whereas a strict detection is a more tenuous link.

(I'm working from my latin here – I don't know Spanish, so do correct me if I'm wrong)

Reply:
I seem to have made a mistake here and interchanged the meanings, I'll change it right now.